README.md

```diff
@@ -278,6 +278,8 @@ add_swagger_documentation \
 This value is added to the `authorizations` key in the JSON documentation.
 
 
+#### *security_definitions*:
+This value is added to the `securityDefinitions` key in the JSON documentation.
 
```

Comment:
maybe adding the spec [#security-definitions-object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#security-definitions-object) link,
so that the people have an idea